tests/Node/TableNodeTest.php

```diff
@@ -303,18 +307,18 @@ public function testMergeRowsFromTablePassSeveralTablesShouldBeMerged(): void
 
     public function testMergeRowsFromTableWrongHeaderNameExceptionThrown(): void
     {
-        $this->expectException(NodeException::class);
         $table = new TableNode([
             5 => ['id', 'username', 'password'],
             10 => ['42', 'everzet', 'qwerty'],
             13 => ['2', 'antono', 'pa$sword'],
         ]);
-
         $new = new TableNode([
             25 => ['id', 'QWE', 'password'],
             210 => ['242', '2everzet', '2qwerty'],
         ]);
 
+        $this->expectException(NodeException::class);
```

Comment:
Probably out of scope of this PR, but shouldn't we also check the exception message in these cases?

Reply:
Yeah, I think so - might as well do it while we're looking at this?

Reply:
Done!